app/src/main/res/layout/main_fragment_controls.xml

```diff
@@ -0,0 +1,156 @@
+                android:padding="@dimen/space_small"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:srcCompat="@drawable/ic_covidlights" />
+
+            <TextView
+                android:id="@+id/change_status_title"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/space_normal"
+                android:text="@string/main_change_status"
+                android:textAppearance="?attr/textAppearanceSubtitle1"
+                android:textColor="@color/text"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@+id/change_status_icon"
+                app:layout_constraintTop_toTopOf="@id/change_status_icon" />
+
+            <TextView
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:text="@string/main_did_you_test_positive"
+                android:textAppearance="?attr/textAppearanceBody1"
+                android:textColor="@color/text_button_secondary"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="@id/change_status_title"
+                app:layout_constraintTop_toBottomOf="@+id/change_status_title" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="@dimen/space_normal">
+
+            <ImageView
+                android:id="@+id/share_proximity_icon"
+                android:layout_width="40dp"
+                android:layout_height="40dp"
+                android:layout_marginTop="26dp"
+                android:layout_marginBottom="26dp"
+                android:backgroundTint="@color/main_control_bg"
+                android:background="@drawable/ic_circle"
+                android:padding="@dimen/space_small"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintLeft_toLeftOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                app:srcCompat="@drawable/ic_bluetooth_notify" />
+
+            <TextView
+                android:id="@+id/share_proximity_title"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/space_normal"
+                android:text="@string/main_share_proximity"
+                android:textAppearance="?attr/textAppearanceSubtitle1"
+                android:textColor="@color/text"
+                app:layout_constraintEnd_toEndOf="parent"
```

Comment:
You missed this comment except part 😋 https://github.com/covilights/AndroidCoviLights/pull/21#discussion_r413233708
```suggestion
                app:layout_constraintEnd_toStartOf="@+id/share_proximity_switch"
```